### ISSUE TITLE
Control on changing near and plane parameters in python api.

### ISSUE THREAD

**Issue by manas-avi** [intel-isl/Open3D#766]
**Is your feature request related to a problem? Please describe.**
Rendered images are getting clipped out at certain positions. 

**Describe the solution you'd like**
Control on near and far plane parameters in python api like we have for field of view in python api.

**Comment by syncle** [comment]
Hello @manas-avi, sorry I haven't checked out this issue for some time. Could you share minimal code snippet so that I can reproduce the issue?

**Comment by manas-avi** [comment]
```
import open3d as o3d
import numpy as np
import pdb

# ------ MESH ------
mesh = o3d.read_triangle_mesh("std.ply")
# estimate normals
mesh.compute_vertex_normals()

# color (only for visual effect)
color = [234, 192, 134]
color = [c/255 for c in color]
mesh.paint_uniform_color(color)

# ----- VISUALIZE -----

# define visualizer
vis = o3d.Visualizer()
vis.create_window(
        window_name="template mesh",
        width=640,
        height=480,
        left=250,
        right=250
        )
# center the mesh
vertices = np.asarray(mesh.vertices)
mean_vertex = np.mean(vertices)
vertices = vertices - mean_vertex

# define camera parameters
view_control = vis.get_view_control()
camera_intrinsic = o3d.PinholeCameraIntrinsic(640, 480, 525, 525, 320-0.5, 240-0.5)
camera_extrinsic = np.array(
        [
            [1, 0, 0, 0],
            [0, 1, 0, 0],
            [0, 0, 1, 0.0],
            [0, 0, 0, 1]
            ]
        )
vis.add_geometry(mesh)
# add data to visualizer
view_control.convert_from_pinhole_camera_parameters(
        camera_intrinsic, 
        camera_extrinsic
        )
view_control.scale(1)  # sets zoom to 1

# translate the mesh away from camera which is at (0,0,0)
initial_distance_from_camera = 40
# vertices = vertices + np.array([0,0,distance_from_camera])
# far plane in this setting is somewhere around 100 units away from camera
# and the image completely falls out of the far plane if distance_from_camera > 120
# and there is no control on increasing the far plane
vertices = vertices + np.array([0,0,initial_distance_from_camera])
distance_from_camera = 0.01
while True:

    vertices = vertices + np.array([0,0,distance_from_camera])
    mesh.vertices = o3d.Vector3dVector(vertices)    
    # render
    vis.update_geometry()
    flag_window_open = vis.poll_events()
    if not flag_window_open:
        break
    
vis.destroy_window()

```

credits for the mesh file used - John Burkardt
mesh file can be found at - https://people.sc.fsu.edu/~jburkardt/data/ply/ant.ply

As you increase the distance of the object from the camera it starts to fade off and then ultimatley falls out of the far plane thus getting clipped. Can we get a control on setting the near and far plane for the camera in python api?

**Comment by manas-avi** [comment]
import open3d as o3d
import numpy as np
import pdb

# ------ MESH ------
mesh = o3d.read_triangle_mesh("std.ply")
# estimate normals
mesh.compute_vertex_normals()

# color (only for visual effect)
color = [234, 192, 134]
color = [c/255 for c in color]
mesh.paint_uniform_color(color)

# ----- VISUALIZE -----

# define visualizer
vis = o3d.Visualizer()
vis.create_window(
        window_name="template mesh",
        width=640,
        height=480,
        left=250,
        right=250
        )
# center the mesh
vertices = np.asarray(mesh.vertices)
mean_vertex = np.mean(vertices)
vertices = vertices - mean_vertex

# define camera parameters
view_control = vis.get_view_control()
camera_intrinsic = o3d.PinholeCameraIntrinsic(640, 480, 525, 525, 320-0.5, 240-0.5)
camera_extrinsic = np.array(
        [
            [1, 0, 0, 0],
            [0, 1, 0, 0],
            [0, 0, 1, 0.0],
            [0, 0, 0, 1]
            ]
        )
vis.add_geometry(mesh)
# add data to visualizer
view_control.convert_from_pinhole_camera_parameters(
        camera_intrinsic, 
        camera_extrinsic
        )
view_control.scale(1)  # sets zoom to 1

# translate the mesh away from camera which is at (0,0,0)
initial_distance_from_camera = 40
# vertices = vertices + np.array([0,0,distance_from_camera])
# far plane in this setting is somewhere around 100 units away from camera
# and the image completely falls out of the far plane if distance_from_camera > 120
# and there is no control on increasing the far plane
vertices = vertices + np.array([0,0,initial_distance_from_camera])
distance_from_camera = 0.01
while True:

    vertices = vertices + np.array([0,0,distance_from_camera])
    mesh.vertices = o3d.Vector3dVector(vertices)    
    # render
    vis.update_geometry()
    flag_window_open = vis.poll_events()
    if not flag_window_open:
        break
    
vis.destroy_window()

credits for the mesh file used - John Burkardt
mesh file can be found at - https://people.sc.fsu.edu/~jburkardt/data/ply/ant.ply

As you increase the distance of the object from the camera it starts to fade off and then ultimatley falls out of the far plane thus getting clipped. Can we get a control on setting the near and far plane for the camera in python api?

**Comment by manas-avi** [comment]
Sorry for the inconvenience, I mistakenly closed the issue.

**Comment by germanros1987** [comment]
Fixed in #1122 .
